lib/govuk_sidekiq/api_headers.rb

```diff
@@ -22,8 +22,8 @@ def call(_worker_class, job, _queue, _redis_pool)
 
       def header_arguments
         {
-          authenticated_user: GdsApi::GovukHeaders.headers[:x_govuk_authenticated_user],
-          request_id: GdsApi::GovukHeaders.headers[:govuk_request_id],
+          "authenticated_user" => GdsApi::GovukHeaders.headers[:x_govuk_authenticated_user],
```

Comment:
Could you add an explanation to the commit message explaining why we needed to do this, as context will get lost over time.

Reply:
done!